src/containers/shared/components/Transaction/AMM/AMMBid/Simple.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../../SimpleRow'
+import { TransactionSimpleProps } from '../../types'
+import { localizeBalance } from '../../../../utils'
+import { Account } from '../../../Account'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { accountID, bidMin, bidMax, authAccounts } = data.instructions
+  const localizedBidMin = localizeBalance(bidMin)
+  const localizedBidMax = localizeBalance(bidMax)
+
+  return (
+    <>
+      {accountID && (
+        <SimpleRow label={t('account_id')} data-test="account_id">
```

Comment:
This label should be something like "AMM Account ID"

Reply:
changed it.